spec/administrate/views/fields/select/_edit_spec.rb

```diff
@@ -9,6 +9,7 @@
       attribute: :email_subscriber,
       data: false,
       selectable_options: [true, false, nil],
+      include_blank_option: false,
```

Comment:
Wrong option name?

```suggestion
      include_blank: false,
```

Reply:
🤦 yeah, thanks

Reply:
just kidding, this is correct. The option itself is named `include_blank`, but the getter on the field is `include_blank_option`. Similar to BelongTo field: https://github.com/thoughtbot/administrate/blob/main/lib/administrate/field/belongs_to.rb#L30-L32

Reply:
Ah, you are right. This is a double and not the real thing, therefore this is a literal method name to stub, as opposed to a constructor argument. Sorry!